resources.py

```diff
@@ -128,4 +128,30 @@ def refresh_tokens():
         token = create_token(user_id)
         return jsonify(token), 200
     except:
-        return jsonify({'error':'invalid token'}), 400
\ No newline at end of file
+        return jsonify({'error':'invalid token'}), 400
+
+@resources.route('/documents', methods=['POST'])
+def create_documents():
```

Comment:
rename: upsert_documents

Reply:
function should handle both creating new documents, and updating existing documents (if the incoming payload comes in with an 'id' parameter)